src/api-reference/invoice/v3.purchase-order-receipt.markdown

```diff
@@ -39,7 +39,7 @@ Name|Type|Format|Description
 
 ```json
 {
-  "Deleted": false,
+  "Deleted": "false",
```

Comment:
Is this a String that's using boolean logic?
If not, the boolean value should not be within quotation marks.

Reply:
Fixed in 51a69ac